src/Exporter/Plugin/ChannelResourcePlugin.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FriendsOfSylius\SyliusImportExportPlugin\Exporter\Plugin;
+
+use Sylius\Component\Addressing\Model\ZoneInterface;
+use Sylius\Component\Core\Model\ChannelInterface;
+use Sylius\Component\Core\Model\ShopBillingDataInterface;
+use Sylius\Component\Currency\Model\CurrencyInterface;
+use Sylius\Component\Locale\Model\LocaleInterface;
+
+final class ChannelResourcePlugin extends ResourcePlugin
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function init(array $idsToExport): void
+    {
+        parent::init($idsToExport);
+
+        /** @var ChannelInterface $resource */
+        foreach ($this->resources as $resource) {
+            if (null !== $resource->getShopBillingData()) {
+                $this->addShopBillingData($resource);
+            }
+
+            if (null !== $resource->getBaseCurrency()) {
+                $this->addBaseCurrency($resource);
+            }
+
+            if (null !== $resource->getDefaultLocale()) {
+                $this->addDefaultLocale($resource);
+            }
+
+            if (null !== $resource->getDefaultTaxZone()) {
+                $this->addDefaultTaxZone($resource);
+            }
+        }
+    }
+
+    private function addShopBillingData(ChannelInterface $channel): void
+    {
+        /** @var ShopBillingDataInterface $shopBillingData */
+        $shopBillingData = $channel->getShopBillingData();
+
+        $this->addDataForResource($channel, 'Postcode', $shopBillingData->getPostcode());
```

Comment:
Can we move it to an array? The same way we do it for translations and product option values:

```json
{
  "Code": "US_WEB",
  "ShopBillingData": {
    "City": "New York"
  }
}
```